src/main/java/no/unit/nva/model/contexttypes/BaseContext.java

```diff
@@ -0,0 +1,18 @@
+package no.unit.nva.model.contexttypes;
+
+import no.unit.nva.model.Level;
+
+public interface BaseContext extends PublicationContext {
```

Comment:
What is a `BaseContext`? What is the difference between a `BaseContext` and a `PublicationContext`?  When is something a `PublicationContext` but not a `BaseContext`? If we move all the methods of `BaseContext` to `PublicationContext` is it a logical error? 

I think a Javadoc documentation exlaining what is this class would help a lot the future developer who will join this project.